lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/statistics/FileEvent.java

```diff
@@ -2,7 +2,10 @@
 
 public enum FileEvent {
 
-	FILE_DOWNLOAD("fileDownloads", "fileDownloadsRecords"), FILE_UPLOAD("fileUploads", "fileUploadsRecords");
+	FILE_DOWNLOAD("fileDownloads", "fileDownloadsRecords"),
+	FILE_UPLOAD("fileUploads", "fileUploadsRecords"),
+	FILE_DOWNLOAD_RECORD("fileDownloadRecords", "fileDownloadsRecords"),
```

Comment:
We do not need an additional type of event, since we are already processing these events.